cmake/flash/CMakeLists.txt

```
@@ -1,20 +1,64 @@
 # SPDX-License-Identifier: Apache-2.0
 
-add_custom_target(runner_yml_props_target)
-
-function(runner_yml_write content)
+function(runners_yaml_append content)
   # Append ${content}\n to a target property which is later evaluated as a
   # generator expression when writing the flash runner yaml file.
   # We define this function here to have access to the `flash` target.
 
   set_property(
-    TARGET         runner_yml_props_target
+    TARGET         runners_yaml_props_target
     APPEND_STRING
     PROPERTY       yaml_contents
     "${content}\n"
     )
 endfunction()
 
+function(get_runners_prop prop out_var)
```

Just wondering, why is the default value `${ARGV2}` not mandatory ?
All calls to this function always provides a default value (as minimum empty string).

```suggestion
function(get_runners_prop prop out_var default_value)
```

---

It's unnecessary generality at this point. I could change it but I was just trying to emulate a kwarg in python.